spec.html

```diff
@@ -612,11 +616,9 @@ <h1>
 					including `HTMLElement`, `console`, `localStorage`, `fetch`, etc.
 				</p>
 			</emu-note>
-			<emu-note type=editor>
+			<emu-note>
 				<p>
-					The ShadowRealm constructor (<emu-xref href="#sec-shadowrealm"></emu-xref>)
-					creates a new global object as an ordinary object. This
-					means all properties from the global object are deletable.
+					The host is expected to create global object whose properties are all deletable.
```

Comment:
```suggestion
					The host is expected to create a global object whose properties are all deletable.
```

Although I think this note is redundant, the paragraph above already states:

> The host may use this hook to add properties to the ShadowRealm's global object. Those properties must be configurable.

...and configurable means deletable.

Reply:
@linusg with historical facts from Ecmascript and Web, it may seem obvious, but not necessarily. Hence @mhofman - along with the proposal champions - emphasizing on the Global being an ordinary object.